python/graphscope/client/session.py

```diff
@@ -778,7 +776,12 @@ def _connect(self):
                 self._pod_name_list,
                 self._config_params["num_workers"],
                 self._config_params["k8s_namespace"],
-            ) = self._grpc_client.connect()
+            ) = self._grpc_client.connect(
+                cleanup_instance=False if self._config_params["addr"] else True,
```

Comment:
You can use `bool(self._config_params["addr"])`.

Reply:
Done